src/libcrun/kontain_nonkmexec.c

```diff
@@ -0,0 +1,343 @@
+/*
+ * crun - OCI runtime written in C
+ *
+ * Copyright (C) 2020 Kontain Inc.
+ * crun is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * crun is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with crun.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <pthread.h>
+#include <string.h>
+#include <time.h>
+#include <sys/queue.h>
+#include <regex.h>
+#include <stdlib.h>
+#include <errno.h>
+#include "utils.h"
+#include "sha256.h"
+
+/*
+ * What is going on in here?
```

Comment:
please drop the question - it adds nothing to the comment

Reply:
Removed.